app/views/docs/upgrade.phtml

```diff
@@ -78,5 +78,9 @@ docker compose exec appwrite migrate</code></pre>
 </div>
 
 <p>The data migration can take longer depending on the amount of data your Appwrite instance contains. The Appwrite migration command uses multi-threading to speed up the process, meaning that adding more CPU cores can help speed up the process.</p>
-
-<p>Once the migration process has completed successfully, you're all set to use the latest version of Appwrite!</p>
\ No newline at end of file
+    
+<p>Once the migration process has completed successfully, you're all set to use the latest version of Appwrite!</p>
+    
+<h3><a href="/docs/upgrade#upgrade-sdk" id="upgrade-sdk">Upgrade SDK</a></h3>
+    
+<p>After upgrading the Appwrite server, ensure that the SDKs used by your applications are compatible with the new Appwrite server version.</p>
```

Comment:
@gewenyu99 maybe we should also add a hint on how to check if the SDK is compatible?

Reply:
Hmmm, yes. 
@zwarag let's add a line that suggests the user to check their SDK version. I think we should suggest they upgrade the SDKs to the latest version if the user is migrating to the latest version. They should check the change log for each SDK on Github for version-specific compatibility.

https://github.com/appwrite/sdk-for-web/releases

Reply:
@gewenyu99 sure, Thx for the feedback. I'll update the PR soon.

Reply:
@gewenyu99 I've updated the commit. If there is anything I can do to improve it. Let me know.